study.md

```diff
@@ -135,13 +145,15 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.select{|item| item%2 !== 0 }
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+array.filter(function(value){
+  return value % 3 == 0;
```

Comment:
This will filter numbers that aren't multiples of 3.  Also, you should be using the strict equality operator (`===`).